metrics/metrics.go

```diff
@@ -178,6 +185,7 @@ func (m *shadowsocksMetrics) AddClosedTCPConnection(clientLocation, accessKey, s
 	m.tcpClosedConnections.WithLabelValues(clientLocation, status, accessKey).Inc()
 	m.tcpConnectionDurationMs.WithLabelValues(clientLocation, status, accessKey).Observe(duration.Seconds() * 1000)
 	m.timeToCipherMs.WithLabelValues("tcp", clientLocation, accessKey).Observe(timeToCipher.Seconds() * 1000)
+	m.tcpProbes.WithLabelValues(clientLocation, status).Observe(float64(data.ClientProxy))
```

Comment:
This is effectively dataBytes for tcp, c>p. It doesn't mean it's a probe.

How do we distinguish the probes? How can we define a probe? Is this a good definition? "All connections that fail authentication (ERR_CIPHER) or that times out before authentication".

What status are we getting for the timeout before authentication?

Reply:
For short probes, it could be useful to differentiate those that timeout from those that don't (because they sent a FIN). In the future, even classify them (e.g. TLS, HTTP, ...) This could justify have a special histogram only for probes. But if we have a histogram only for probes, we should remove the status and only register probes in it.